NISTIR_Metadata.md

```diff
@@ -26,7 +26,7 @@ Attribute schema metadata provide information that is applicable to the attribut
 | **Allowed Values** | A defined set of allowed values for the attribute         |             None                                  |                                                    
 |**Format**| A defined format in which the attribute will be expressed| None|
 | **Verification Frequency** |The frequency at which the AP will re-verify the attribute| None |
-| **Consent** | The type of consent obtained | None |
+| **Data Processing** | Type of requirement | None |

```

Comment:
Suggest changing the description to "Describes the basis for processing attributes and attribute values.